Userland/Libraries/LibJS/Runtime/ArrayBuffer.cpp

```diff
@@ -57,6 +57,23 @@ bool ArrayBuffer::is_resizable_array_buffer() const
     return m_max_byte_length.has_value();
 }
 
+// 1.1.7 HostResizeArrayBuffer ( buffer, newByteLength ), https://tc39.es/proposal-resizablearraybuffer/#sec-hostresizearraybuffer
+ThrowCompletionOr<bool> ArrayBuffer::host_resize_array_buffer(GlobalObject& global_object, size_t new_byte_length)
+{
+    // The host-defined abstract operation HostResizeArrayBuffer takes arguments buffer (an ArrayBuffer) and newByteLength (a non-negative integer).
+    // The host-defined abstract operation HostResizeArrayBuffer takes arguments buffer (an ArrayBuffer object) and newByteLength.
```

Comment:
This sentence appears twice in the spec, reminder for myself to file an issue :smile: